app/core/custom_app.py

```diff
@@ -61,3 +45,25 @@ def create_configuration_token(
         "iat": datetime.timestamp(issued_at.replace(microsecond=0)),  # No microseconds
     }
     return jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
+
+
+def decode_configuration_token(token: str) -> list:
```

Comment:
What are we returning a list of? `list[int]` or `list[float]` or list[str]` etc?